bin/python/ripple/util/Sign.py

```diff
@@ -22,9 +23,9 @@
     check <key>
         Check an existing key for validity.
 
-    sign <sequence> <validator-public> <master-secret>
+    sign <sequence> <validator-public-key> <validator-private-key> <master-secret>
         Create a new signed manifest with the given sequence
-        number, validator public key, and master secret key.
+        number, validation public key, and master public key.
```

Comment:
I'm trying to follow the directions in `doc/manifest-tool-guide.md` and I'm getting lost.  There are a lot of keys flying by here.  I'd like to suggest that, as far as humanly possible, we always use *exactly* the same name for a give type of key.  Here are the names I think we can't escape because they are in the config or are otherwise human facing:

- The `validator_key`.
- The `master_secret`.
- The `validation_public_key`.  Also called the "ephemeral public key".
- The `validation_seed`.  Also called the "ephemeral secret key".

If we can limit ourselves to those 6 names and always use the same spelling (in both C++ and Python, *especially* in the UI visible text, and please don't replace `_` with `-` or a space) I think we'll end up way ahead.

What I'd like to suggest is that you make a pass through `doc/manifest-tool-guide.md` and update it to match the new process.  After you've done that I'll attempt to follow the process using the updated doc.  (By the way, there's an odd sentence fragment on line 67 of the doc: "sequence numbers are be explained below".  You may want to fix that as well.)